doc/netplan.md

```diff
@@ -380,15 +380,20 @@ wpasupplicant installed if you let the ``networkd`` renderer handle wifi.
 
      ``password`` (scalar)
      :    Enable WPA2 authentication and set the passphrase for it. If not
-          given, the network is assumed to be open. Other authentication modes
-          are not currently supported.
+          given, the network is assumed to be open.
 
      ``mode`` (scalar)
      :    Possible access point modes are ``infrastructure`` (the default),
           ``ap`` (create an access point to which other devices can connect),
           and ``adhoc`` (peer to peer networks without a central access point).
           ``ap`` is only supported with NetworkManager.
 
+     ``key_mgmt`` (scalar)
+     :    Possible key_mgmt values are WPA-PSK and WPA-EAP. Optional.
+
+     ``identity`` (scalar)
+     :    Provides the identity for authenticating with WPA-EAP.
+
```

Comment:
This should be squashed in to the previous comment so that the build succeeds at each point (which is important for bisectability).

Also those descriptions are really terse, and should at the very least mention that they are for WPA2 Enterprise and not required for WPA2 Personal.